internal/prep/prep_stateroot_setup_job.go

```diff
@@ -0,0 +1,167 @@
+)
+
+func GetStaterootSetupJob(ctx context.Context, c client.Client, log logr.Logger) (*batchv1.Job, error) {
+	job := &batchv1.Job{}
+	if err := c.Get(ctx, types.NamespacedName{Name: JobName, Namespace: common.LcaNamespace}, job); err != nil {
+		return job, err //nolint:wrapcheck
+	}
+
+	log.Info("Got stateroot setup job", "namespace", job.Namespace, "name", job.Name)
+
+	return job, nil
+}
+
+func LaunchStaterootSetupJob(ctx context.Context, c client.Client, ibu *v1alpha1.ImageBasedUpgrade, scheme *runtime.Scheme, log logr.Logger) (*batchv1.Job, error) {
+	job, err := constructJobForStaterootSetup(ctx, c, ibu, scheme, log)
+	if err != nil {
+		return nil, fmt.Errorf("failed to render job: %w", err)
+	}
+
+	if err := c.Create(ctx, job); err != nil {
+		return nil, err //nolint:wrapcheck
+	}
+
+	log.Info("Successfully created job", "job", job.Name)
+	return job, nil
+}
+
+func constructJobForStaterootSetup(ctx context.Context, c client.Client, ibu *v1alpha1.ImageBasedUpgrade, scheme *runtime.Scheme, log logr.Logger) (*batchv1.Job, error) {
+	log.Info("Getting lca deployment to configure stateroot setup job")
+	lcaDeployment := appsv1.Deployment{}
+	if err := c.Get(ctx, types.NamespacedName{Namespace: common.LcaNamespace, Name: "lifecycle-agent-controller-manager"}, &lcaDeployment); err != nil {
+		return nil, fmt.Errorf("failed to get lifecycle-agent-controller-manager deployment: %w", err)
+	}
+
+	log.Info("Selecting 'manager' from LCA deployment", "deployment", lcaDeployment.Name)
+	manager, ok := getManagerContainer(lcaDeployment)
+	if !ok {
+		return nil, fmt.Errorf("no 'manager' container found in deployment")
+	}
+
+	var backoffLimit int32 = 0
+	job := &batchv1.Job{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      JobName,
+			Namespace: common.LcaNamespace,
+			Annotations: map[string]string{
+				"app.kubernetes.io/name": JobName,
+			},
+		},
+		Spec: batchv1.JobSpec{
+			BackoffLimit: &backoffLimit,
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{
+						common.WorkloadManagementAnnotationKey: common.WorkloadManagementAnnotationValue,
+					},
+				},
+				Spec: corev1.PodSpec{
+					Containers: []corev1.Container{
+						{
+							Name:            JobName,
+							Image:           manager.Image,
+							ImagePullPolicy: manager.ImagePullPolicy,
+							Command:         []string{"lca-cli", "stateroot-setup-ibu"},
+							Env:             manager.Env,
+							EnvFrom:         manager.EnvFrom,
+							SecurityContext: manager.SecurityContext, // this is needed for podman
+							VolumeMounts:    manager.VolumeMounts,
+							Resources:       manager.Resources,
+						},
+					},
+					HostPID:            lcaDeployment.Spec.Template.Spec.HostPID, // this is needed for rpmostree
+					ServiceAccountName: lcaDeployment.Spec.Template.Spec.ServiceAccountName,
+					RestartPolicy:      corev1.RestartPolicyNever,
+					Volumes:            lcaDeployment.Spec.Template.Spec.Volumes,
```

Comment:
Need to add requests, see precache job, https://github.com/openshift-kni/lifecycle-agent/blob/main/internal/precache/helper.go#L213

Reply:
Yep it's right under the containers list (L93). We are grabbing everything from LCA basically....when rendered i see 
```
        resources:
          requests:
            cpu: 100m
            memory: 20Mi
```